apps/nfid-frontend/src/features/sdk/request-transfer/index.tsx

```diff
@@ -200,7 +200,7 @@ export const RequestTransfer: React.FC<IRequestTransferProps> = ({
               new Promise(async (resolve) => {
                 try {
                   console.debug("RequestTransfer", { derivationOrigin, origin })
-                  if (!isPresentInStorage(derivationOrigin || origin))
+                  if (!(await isPresentInStorage(derivationOrigin || origin)))
```

Comment:
would it make sense to avoid this await? fetchProfile is the first request we make to API. I guess we can move it to static cache.